easybuild/easyconfigs/f/FastANI/FastANI-1.1-foss-2018b.eb

```diff
@@ -0,0 +1,41 @@
+# easybuild easyconfig
+#
+# John Dey jfdey@fredhutch.org
+
+easyblock = 'ConfigureMake'
+
+name = 'FastANI'
+version = '1.1'
+
+homepage = "http://www.iodbc.org/"
+description = """FastANI is developed for fast alignment-free computation of
+ whole-genome Average Nucleotide Identity (ANI). ANI is defined as mean
+ nucleotide identity of orthologous gene pairs shared between two microbial
+ genomes. FastANI supports pairwise comparison of both complete and draft
+ genome assemblies."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = ['https://github.com/ParBLiSS/FastANI/archive']
+sources = ['v%(version)s.zip']
+patches = ['FastANI-1.1-memcpy.patch']
+checksums = [
+    'ffb776ddfa2595bba5611b0b017aad5089f9a19ea020c7d708d09c628bb926b3',  # v1.1.zip
+    '12a5ab7514f1c04f6f814ba7b2c11a02640d82d494c53f63a51483f6987da9b6',  # FastANI-1.1-memcpy.patch
+]
+
+preconfigopts = 'autoconf && '
+
+builddependencies = [('Autoconf', '2.69')]
+
+dependencies = [
+    ('GSL', '2.5'),
+    ('zlib', '1.2.11'),
+]
+
+sanity_check_paths = {
+    'files': ['bin/fastANI'],
+    'dirs': ['bin'],
```

Comment:
@fizwit There's little point in checking for a non-empty `bin` directory is you're already checking for `bin/fastANI`, so I'd drop the `dirs` entry:

```python
sanity_check_paths = {
    'files': ['bin/fastANI'],
    'dirs': [],
}